.github/workflows/build.yml

```diff
@@ -5,32 +5,44 @@ on: [push, pull_request]
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ubuntu-latest, windows-latest]
         python-version: [3.7, 3.8]
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
+    - name: Set up Miniconda with Python version ${{ matrix.python-version }}
+      uses: goanpeca/setup-miniconda@v1
       with:
+        auto-update-conda: true
+        channels: conda-forge
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
+      shell: bash -l {0}
```

Comment:
Why is this necessary?

Reply:
It's to ensure the conda env is activated. See https://github.com/marketplace/actions/setup-miniconda#important

I can add a comment to document this.

Reply:
Oh, this is why internally we use:

```
    - uses: s-weigand/setup-conda@v1
      with:
        python-version: 3.8
        conda-channels: conda-forge
```

that makes it easier to use the `base` env.